### PR TITLE
[RESTEASY-3383] Ensure the RESTEasy modules are copied even when laye…

### PR DESCRIPTION
**Pull request by jamezp**
…rs are being used or inherit-package is set to false.

https://issues.redhat.com/browse/RESTEASY-3383

### CODE REVIEW

File: resteasy-feature-pack/galleon-preview-feature-pack/pom.xml [hunk 1]
```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <excludes>
+                                        <include>configs/**/model.xml</include>
```
Comment (jimma) [inline resteasy-feature-pack/galleon-preview-feature-pack/pom.xml:102]: Should it be <exclude> ?

File: resteasy-feature-pack/galleon-feature-pack/pom.xml [hunk 2]
```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <excludes>
```
Comment (jfdenise) [inline resteasy-feature-pack/galleon-feature-pack/pom.xml:101]: Why do you need to exclude it?

File: resteasy-feature-pack/galleon-preview-feature-pack/pom.xml (hunk 1 above)
Comment (jamezp) [reply resteasy-feature-pack/galleon-preview-feature-pack/pom.xml:102]: No, it should be `includes`. I just copied a code snippet and didn't change it all :)

File: resteasy-feature-pack/galleon-feature-pack/pom.xml (hunk 2 above)
Comment (jamezp) [reply resteasy-feature-pack/galleon-feature-pack/pom.xml:101]: Just a copy/paste/typo error. I'll change it to `<includes>`.

File: resteasy-feature-pack/galleon-feature-pack/pom.xml [hunk 3]
```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <includes>
+                                        <exclude>configs/**/model.xml</exclude>
```
Comment (jfdenise) [inline resteasy-feature-pack/galleon-feature-pack/pom.xml:102]: @jamezp , we still have an exclude? (I suppose could be named anything).

Comment (jamezp) [reply resteasy-feature-pack/galleon-feature-pack/pom.xml]: Oh man, I need to pay more attention in the mornings haha. Thank you @jfdenise

**Review by jfdenise — APPROVED** [review]

**Pull request merged**